ashley/src/main/kotlin/ktx/ashley/engines.kt

```diff
@@ -74,6 +75,22 @@ inline fun Engine.entity(configure: EngineEntity.() -> Unit = {}): Entity {
   return entity
 }
 
+/**
+ * Quick [EntitySystem] retrieval.
+ *
+ * @return the [EntitySystem] of the given type. May be null if it does not exist.
+ * @see Engine.getSystem
+ */
+inline fun <reified T : EntitySystem> Engine.getSystem(): T? = getSystem(T::class.java)
```

Comment:
I could see it being more useful if it returned an `EntitySystem` or threw an exception. You usually know which systems you add to an engine.

`get` could remain nullable, so we support both options and these extensions are not redundant.

Reply:
Totally agree. When you get a system you usually know that it is there and you want to do something with it (except you messed up something ;) ).

I will throw an exception.